modules/actions/reverse.js

```diff
@@ -126,6 +130,10 @@ export function actionReverse(entityID, options) {
             for (var key in node.tags) {
                 tags[reverseKey(key)] = reverseValue(key, node.tags[key], node.id === entityID);
             }
+
+            // for directionless traffic signs, the first flip just adds the direction tag
+            if (isDirectionlessTrafficSign(tags)) tags.direction = 'forward';
```

Comment:
This method name is a little confusing; I thought at first that it would apply to `traffic_sign=*` nodes, which could potentially benefit from this improvement when mapped as a vertex along a roadway. However, a `traffic_sign` node can also be a standalone node, in which case `direction` needs to be an angle, not `forward` or `backward`.

Besides traffic signs, the same improvement would benefit `highway=traffic_signals` (when mapped at the stop line) and `traffic_calming=bump`/`hump`/`table`.